Packs/Slack/Integrations/SlackV3/SlackV3.yml

```diff
@@ -106,6 +106,12 @@ configuration:
   required: false
   type: 0
   additionalinfo: Only supported in Demisto V6.1 and above.
+- display: ''
```

Comment:
no display?
also we probably do need to add additional info that say when we need it ad direct to the docs.

Reply:
I think it's handled by the `displayPassword` field no?